index.js

```diff
@@ -35,36 +37,80 @@ module.exports = {
   included: function(app) {
     patchEmberApp(app);
 
-    var mode;
-
-    if (isFastBoot()) {
-      process.env[APP_NAME_KEY] = app.name;
-      app.options.autoRun = false;
+    // We serve the index.html from fastboot-dist, so this has to apply to both builds
+    app.options.storeConfigInMeta = false;
```

Comment:
Without this ember-cli creates a module for the app which uses a jQuery selector to find a meta tag in the `index.html` (which the fastboot started app doesn't have access to).  We need to change the way ember-cli generates the config and emits the index.html. I have generally been working on it, but have fallen behind a bit :crying_cat_face:.